app/src/main/res/layout/fragment_settings_basic.xml

```diff
@@ -11,60 +13,137 @@
                 android:id="@+id/editText_settings_name"
                 android:maxLines="1"
-                android:textSize="@dimen/docText_18"
-                android:singleLine="true"/>
+                android:textSize="18dp"
+                android:singleLine="true"
+                android:gravity="end"
+                android:background="@color/transparent"
+                android:selectAllOnFocus="false"
+                android:paddingEnd="10dp" />
         </LinearLayout>
 
         <LinearLayout
             android:layout_width="match_parent"
-            android:layout_height="@dimen/spinner_60"
+            android:layout_height="1dp"
+            android:background="@color/grey">
+        </LinearLayout>
+
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="50dp"
             android:orientation="horizontal"
-            android:layout_marginTop="@dimen/margin_20"
-            android:padding="5sp"
-            android:weightSum="1">
+            android:weightSum="1"
+            android:layout_gravity="left"
+            android:padding="5dp">
             <TextView
                 android:layout_height="wrap_content"
                 android:layout_width="100sp"
                 android:text="@string/sexSettings"
                 android:fontFamily="sans-serif"
-                android:textSize="@dimen/docText_18"
-                android:textColor="@color/black"
-                android:layout_gravity="center" />
-
-            <Spinner
-                android:layout_width="@dimen/spinner_150"
-                android:layout_height="@dimen/spinner_50"
-                android:id="@+id/spnSex_Settings"
-                android:gravity="center"
-                android:fontFamily="sans-serif"
-                android:textColor="@color/black"
-                android:layout_weight="1.10" />
+                android:textSize="17dp"
+                android:textColor="@color/material_grey_600"
+                android:layout_gravity="center"
+                android:paddingStart="5dp" />
+
+
+            <LinearLayout
+                android:layout_width="fill_parent"
+                android:layout_height="50dp"
+                android:gravity="end"
+                android:paddingBottom="5dp">
+                <Spinner
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/spnSex_Settings"
+                    android:fontFamily="sans-serif"
+                    android:textDirection="rtl"
+                    android:gravity="end"
+                    android:paddingEnd="5dp"
+                    android:paddingRight="5dp" />
+            </LinearLayout>
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="New Text"
```

Comment:
If this is a dummy text then remove it. Otherwise move this string to resource file.